src/test/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeatTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
```

Comment:
I have written this class to verify if Java 17 feature works as a POC. This change is not required as part of this ticket

Reply:
done